e2e/cypress/integration/plugins/marketplace/marketplace_spec.js

```diff
@@ -152,10 +152,10 @@ describe('Plugin Marketplace', () => {
             cy.get('#error_bar').should('not.be.visible');
 
             // * search should be visible
-            cy.get('#searchMarketplaceTextbox').should('be.visible');
+            cy.findByPlaceholderText('Search Plugins').should('be.visible').click();
```

Comment:
if we are acting on it, it is no longer an assertion, we should update the comment above